components/paging/pager-load-more.js

```diff
@@ -0,0 +1,155 @@
+import '../colors/colors.js';
+import '../loading-spinner/loading-spinner.js';
+import { css, html, LitElement } from 'lit';
+import { buttonStyles } from '../button/button-styles.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { findComposedAncestor } from '../../helpers/dom.js';
+import { FocusMixin } from '../../mixins/focus-mixin.js';
+import { FocusVisiblePolyfillMixin } from '../../mixins/focus-visible-polyfill-mixin.js';
+import { getFirstFocusableDescendant } from '../../helpers/focus.js';
+import { labelStyles } from '../typography/styles.js';
+import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
+
+const nativeFocus = document.createElement('div').focus;
+
+/**
+ *  A pager component for load-more paging.
+ * @fires d2l-pager-load-more - Dispatched when the user clicks the load-more button. Consumers must call the provided "complete" method once items have been loaded.
+ */
+class LoadMore extends FocusMixin(FocusVisiblePolyfillMixin(LocalizeCoreElement(LitElement))) {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether there are more items that can be loaded.
+			 * @type {boolean}
+			 */
+			hasMore: { type: Boolean, attribute: 'has-more', reflect: true },
+			/**
+			 * Total number of items. If not specified, neither it nor the count of items showing will be displayed.
+			 * @type {number}
+			 */
+			itemCount: { type: Number, attribute: 'item-count', reflect: true },
+			/**
+			 * The number of additional items to load.
+			 * @type {number}
+			 */
+			pageSize: { type: Number, attribute: 'page-size', reflect: true },
+			/**
+			 * The number of items showing. Assigned by PageableMixin.
+			 * @ignore
+			 */
+			itemShowingCount: { type: Number },
+			_loading: { state: true }
```

Comment:
I think these still have a `type`?
```suggestion
			_loading: { state: true, type: Boolean }
```

Reply:
I don't _think_ so. The [Lit docs](https://lit.dev/docs/components/properties/#internal-reactive-state) only show examples that specify `state`, and they say the only other valid option is `hasChanged`.

Reply:
Cool yep, my misunderstanding!